templates/index.mustache

```diff
@@ -0,0 +1,70 @@
+<?php
+
+function populate_site_info() {
```

Comment:
Please add comments in this file where needed. I believe this will be copied paste from mustache template as .php file and it won't have any comment.